src/backupNg/backupNg.spec.js

```diff
@@ -169,5 +169,43 @@ describe("backupNg", () => {
       });
       expect(log.warnings).toMatchSnapshot();
     });
+
+    it("fails trying to run a backup job with a VM without disks", async () => {
+      const { id: template } = find(xo.objects.all, { type: "VM-template" });
+      const vmIdWithoutDisks = await xo.createTempVm({
+        name_label: "XO Test Without Disks",
+        name_description: "Creating a vm without disks",
+        template,
+      });
+
+      const scheduleTempId = randomId();
+      const { id: jobId } = await xo.createTempBackupNgJob({
+        ...defaultBackupNg,
+        schedules: {
+          [scheduleTempId]: {
+            name: "scheduleTest",
+            cron: "0 * * * * *",
+          },
+        },
+        settings: {
+          [scheduleTempId]: { snapshotRetention: 1 },
+        },
+        vms: {
+          id: vmIdWithoutDisks,
+        },
+      });
+
+      const schedule = await xo.getSchedule({ jobId });
+      expect(typeof schedule).toBe("object");
+      await xo.call("backupNg.runJob", { id: jobId, schedule: schedule.id });
+
+      const [log] = await xo.call("backupNg.getLogs", {
+        scheduleId: schedule.id,
+      });
+      expect(log.status).toBe("skipped");
```

Comment:
Can we test more properties using [property-matchers](https://jestjs.io/docs/en/snapshot-testing#property-matchers)?